lighthouse-core/test/lib/asset-saver-test.js

```diff
@@ -78,15 +61,15 @@ describe('asset-saver helper', () => {
       const ssHTMLFilename = 'the_file-0.screenshots.html';
       const ssFileContents = fs.readFileSync(ssHTMLFilename, 'utf8');
       assert.ok(/<!doctype/gim.test(ssFileContents));
-      const expectedScreenshotContent = '{"timestamp":674089419.919';
+      const expectedScreenshotContent = '{"timestamp":668545858.596';
```

Comment:
now really pulling screenshots from the trace instead of having `requestScreenshots` resolve on the saved screenshots, which turned out weren't from this trace :)